packages/vue-components/src/PageNavButton.vue

```diff
@@ -52,9 +35,12 @@ export default {
     },
   },
   mounted() {
-    this.src = window.location.pathname;
-    this.hasIdentifier = document.getElementById('page-nav') !== null;
-    this.hasPageNav = document.getElementById('mb-page-nav') !== null;
+    if (document.getElementById('page-nav') !== null) {
+      this.portalName = 'page-nav';
+    } else if (document.getElementById('mb-page-nav') !== null) {
```

Comment:
As mentioned above, `page-nav` if present will always exist. As `mb-page-nav` will only be present if a page nav is present, we can use this to check if the `page-nav-button` should be shown.

Reply:
reverted temporarily